cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.export;
+
+import com.powsybl.cgmes.conversion.elements.CgmesTopologyKind;
+import com.powsybl.cgmes.conversion.extensions.CgmesSshMetadata;
+import com.powsybl.cgmes.conversion.extensions.CgmesSvMetadata;
+import com.powsybl.cgmes.conversion.extensions.CimCharacteristics;
+import com.powsybl.cgmes.model.CgmesNamespace;
+
+import com.powsybl.iidm.network.Network;
+import org.joda.time.DateTime;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class CgmesExportContext {
+
+    private int cimVersion = 16;
+    private CgmesTopologyKind topologyKind = CgmesTopologyKind.BUS_BRANCH;
+    private DateTime scenarioTime = DateTime.now();
+
+    private ModelDescription svModelDescription = new ModelDescription("SV Model", CgmesNamespace.SV_PROFILE);
+    private ModelDescription sshModelDescription = new ModelDescription("SSH Model", CgmesNamespace.SSH_PROFILE);
+
+    private boolean exportBoundaryPowerFlows = false;
+
+    public static final class ModelDescription {
+
+        private String description = "Model";
+        private int version = 1;
+        private final List<String> dependencies = new ArrayList<>();
+        private String modelingAuthoritySet = "powsybl.org";
+        // TODO Each model may have a list of profiles, not only one
```

Comment:
Can you explain this comment to me?

Reply:
This is a comment relevant if we try to build a generic `ModelDescription` class. In general a single file CGMES XML can contain references to multiple profiles. For example an `EQ` file may contain references to `EquipmentBoundary`, `EquipmentCore`, `EquipmentOperation` (if it is node-breaker), `ShortCircuit`, ...